fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/EraseResourceDAO.java

```diff
@@ -305,9 +305,13 @@ public void deleteFromAllParametersTables(String tablePrefix, long logicalResour
         deleteFromParameterTable(tablePrefix + "_latlng_values", logicalResourceId);
         deleteFromParameterTable(tablePrefix + "_resource_token_refs", logicalResourceId);
         deleteFromParameterTable(tablePrefix + "_quantity_values", logicalResourceId);
+        deleteFromParameterTable(tablePrefix + "_profiles", logicalResourceId);
+        deleteFromParameterTable(tablePrefix + "_tags", logicalResourceId);
         deleteFromParameterTable("str_values", logicalResourceId);
         deleteFromParameterTable("date_values", logicalResourceId);
         deleteFromParameterTable("resource_token_refs", logicalResourceId);
+        deleteFromParameterTable("logical_resource_profiles", logicalResourceId);
+        deleteFromParameterTable("logical_resource_tags", logicalResourceId);
```

Comment:
@michaelwschroeder you mentioned you'd add the new security table deletes here too.

Reply:
Done